docs/application/web/guides/connectivity/download.md

```diff
@@ -85,25 +85,45 @@ To provide the user access to Internet resources, you must learn how to manage d
 
    The `start()` method returns a unique identifier for the download operation.
 
-5. During the download:
+   You can check the status of the download operation by calling the `getState()` method with the download ID:
+
+      ```
+      tizen.download.getState(downloadId);
+      ```
+
+5. During the download operation, you can modify the download state.
+
+    >**Note**
+    >
+    >The download operation gets completed almost instantaneously, depending on download size and connection speed.
 
    1. To pause the download, use the `pause()` method with the download ID:
 
       ```
       tizen.download.pause(downloadId);
       ```
+        Calling the `pause()` method will result in the PAUSED download state only if the previous state was DOWNLOADING or QUEUED.
 
    2. To resume the download, use the `resume()` method with the download ID:
 
       ```
       tizen.download.resume(downloadId);
       ```
+        Calling the `resume()` method will result in the DOWNLOADING or the QUEUED download state only if the previous state was PAUSED, CANCELED or FAILED.
 
    3. To cancel the download, use the `cancel()` method with the download ID:
 
       ```
       tizen.download.cancel(downloadId);
       ```
+        Calling the `cancel()` method will result in the CANCELED download state only if the previous state was DOWNLOADING or QUEUED.
+
+   4. To abandon the download, use the `abandon()` method with the download ID:
+
+      ```
+      tizen.download.abandon(downloadId);
+      ```
+        The abandoned download operation will have its resources freed up, making it impossible to perform any further operations on this download operation.
```

Comment:
@mijin85cho 
Thanks for your valuable suggestions. 
I will definitely take your suggestions as future tense must be avoided as much as possible. 

Reason: 

I kept the original format only because I was not sure that if abandoned download operation has the capability to free its resources and I did not want to rephrase without knowing.

The abandoned download operation will have its resources freed up, making it impossible to perform any further operations on this download operation.

Abandoned download operation may or may not have the capability to free its resources, but some other service may be called while the download is abandoning to free its resources.
So this made me to not to change the tense. 
Since this whole process depends on the abandonment of download. So only when the download will abandon, then this will happen, so the future tense.

Reply:
When `abandon()` method is called with the id of the download operation, the method gives signal to free the operations resources and almost immediately the `abandon()` method is internally provided with feedback if this was done correctly, only after the feedback the `abandon()` method ends without rising exceptions.
So if `abandon()` method has been called and no exceptions was raised, then resources are already freed.

Reply:
@apietraszek 

Thanks for the details, I think @mijin85cho 's sentence is fine then.

The abandoned download operation frees its resources and stops further download operations. 

Kindly let me know about your opinion.

Reply:
Done